lib/capybara/driver/node.rb

```diff
@@ -5,6 +5,24 @@ module Driver
     class Node
       attr_reader :driver, :native, :initial_cache
 
+      NON_BREAKING_SPACE = "\u00a0"
+      LINE_SEPERATOR = "\u2028"
+      PARAGRAPH_SEPERATOR = "\u2029"
```

Comment:
Made some of the unicode characters into named constants to know what each one did, as looking them up takes you away from the code.

Reply:
Why are you putting these in this class? I get that it's a base but it's also used by a number of 3rd party drivers.  These should be defined in any driver specific nodes they are needed/wanted in (or into a module which can be included into those driver specific classes)

Reply:
Mostly base class, yes, considering that Selenium and RackTest both used it. Module makes sense, some sort of whitespace cleaner, can get that done tonight when I get back.